package-lock.json

```diff
@@ -3764,7 +3764,7 @@
       "requires": {
         "esprima": "1.2.2",
         "static-eval": "2.0.2",
-        "underscore": "^1.12.1"
+        "underscore": "1.7.0"
       },
       "dependencies": {
         "esprima": {
```

Comment:
please revert this file.

Reply:
@jianyexi I have reverted the file.

Reply:
@jianyexi Changes requested have been made, could you look over it, please?